frontend/src/components/question-creation/question-elements/ShortAnswerElement.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Button, IconButton, Input, Flex, Box, Spacer } from "@chakra-ui/react";
+import { EditOutlineIcon } from "../../../assets/icons";
+import EditShortAnswerModal from "./modals/short-answer/EditShortAnswerModal";
+
+interface ShortAnswerElementProps {
+  id: string;
+  data: string;
+}
+
+const ShortAnswerElement = ({
+  id,
+  data,
+}: ShortAnswerElementProps): React.ReactElement => {
+  const [
+    showEditShortAnswerModal,
+    setShowEditShortAnswerModal,
+  ] = React.useState(false);
+  return (
+    <Flex width="100%" paddingLeft="6">
+      <Input
+        value={data}
```

Comment:
differs from the Figma designs, but design team has asked to have the answer value shown here